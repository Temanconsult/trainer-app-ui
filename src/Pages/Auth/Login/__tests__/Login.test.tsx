import React from 'react';
import {render} from '@testing-library/react';
import {BrowserRouter as Router} from 'react-router-dom';
import {Login} from '..';

describe('<Login />', () => {
  test('renders login component elements correctly', () => {
    const {container} = render(
      <Router>
        <Login />
      </Router>
    );
    expect(container).toMatchSnapshot();
  });
});
