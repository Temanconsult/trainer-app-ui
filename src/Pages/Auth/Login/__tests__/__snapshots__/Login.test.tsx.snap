// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> renders login component elements correctly 1`] = `
<div>
  <div
    class="h-screen flex flex-row"
  >
    <div
      class="h-full w-1/2 hidden lg:block"
    >
      <div
        class="h-full w-full p-5 flex justify-center items-center"
      >
        <svg>
          login.svg
        </svg>
      </div>
    </div>
    <div
      class="flex flex-col justify-center relative w-full lg:w-1/2 "
    >
      <div
        class="flex justify-center mb-10"
      >
        <svg>
          logo.svg
        </svg>
      </div>
      <span
        class="text-center mb-10 text-black opacity-50 text-base"
      >
        Welcome back! 
        <br
          class="sm:inline lg:hidden"
        />
        <span>
          Please login to your account.
        </span>
      </span>
      <div
        class="flex flex-col justify-center items-center w-full h-64"
      >
        <form
          class="flex flex-col w-10/12 md:w-8/12 h-full"
        >
          <div
            class="input-wrapper my-4 w-full"
          >
            <input
              class="input py-2 border-gray-400 border-b text-xs outline-none"
              name="username"
              placeholder="Username"
              type="text"
            />
          </div>
          <div
            class="input-wrapper my-4 w-full"
          >
            <input
              class="input py-2 border-gray-400 border-b text-xs outline-none"
              name="password"
              placeholder="Password"
              type="password"
            />
          </div>
          <div
            class="flex justify-between mt-6 items-center"
          >
            <div
              class="flex items-center"
            >
              <input
                name="rememberMe"
                type="checkbox"
              />
              <span
                class="ml-2"
              >
                Remember me
              </span>
            </div>
            <a
              class="italic"
              href="/"
            >
              Forgot Password?
            </a>
          </div>
          <div
            class="flex justify-between mt-10 items-center"
          >
            <button
              class="btn text-white bg-custom-primary w-1/2 mr-2"
              type="submit"
            >
              Login
            </button>
            <a
              class="btn text-black border border-custom-primary opacity-75 w-1/2 ml-2"
              href="/signup"
            >
              Sign up
            </a>
          </div>
        </form>
      </div>
      <span
        class="absolute left-0 right-0 bottom-0 p-4 text-xs text-center text-custom-gray"
      >
        Term of use. Privacy policy
      </span>
    </div>
  </div>
</div>
`;
